src/Microsoft.TestPlatform.Extensions.BlameDataCollector/BlameCollector.cs

```diff
@@ -131,11 +135,11 @@ public BlameCollector()
         if (_configurationElement != null)
         {
             var collectDumpNode = _configurationElement[Constants.DumpModeKey];
-            _collectProcessDumpOnTrigger = collectDumpNode != null;
+            _collectProcessDumpOnCrash = collectDumpNode != null;
```

Comment:
Maybe that's outside of the scope of this PR but do we need this bool field? I feel like we could just do the null check here on line 140 and use `_crashDumpType` later on. 

I think the same remark applies for the `_collectProcessDumpOnHang` field.

Reply:
I like having a bool field that enables the whole functionality more than leaning on nullability on a field that selects which dump type should be used. Feels unnecessary to mix two different concepts (enabling it and dump type) into one field, when fields are free. It is also easier to see that we really meant to check if crash dump is enabled even though we are currently in hang dump related method: 

![image](https://user-images.githubusercontent.com/5735905/164161304-bbe3c868-df61-4839-b41f-12f6821a776d.png)


e.g. here where we need to detach procdump before killing testhost.

Reply:
Fine by me.